src/Components/Address/__tests__/useAddressAutocomplete.jest.ts

```diff
@@ -16,10 +16,13 @@ jest.mock("lodash/throttle", () => jest.fn(fn => fn))
 
 let mockFetch: jest.Mock
 
+type HookProps = Parameters<typeof useAddressAutocomplete>
+let hookOptions = { enableSecondarySuggestions: false }
+
 describe("useAddressAutocomplete", () => {
-  const setupHook = (...args: Parameters<typeof useAddressAutocomplete>) => {
-    const result = renderHook(props => useAddressAutocomplete(props), {
-      initialProps: args[0],
+  const setupHook = (...args: HookProps) => {
+    const result = renderHook(props => useAddressAutocomplete(...props), {
+      initialProps: args,
```

Comment:
one thing is i got a little mixed up here since i have to pass two args to the hook now. as a result the way i pass them looks a little weird.